azerothcore/azerothcore-wotlk
# fix(Docker): fix Priority class denied

## CHANGES PROPOSED:
-  set worldserver docker as privileged


## ISSUES ADDRESSED:
- https://github.com/azerothcore/azerothcore-wotlk/issues/92 (only address the issue for docker)



## TESTS PERFORMED:
- Tested on centos 7



## HOW TO TEST THE CHANGES:

- make sure to clear any docker cache (docker system prune -a)
- run a docker-compose up
- verify the process priority class gets set and not showing "Can't set worldserver process priority class, error: Permission denied" any more in the console.

## KNOWN ISSUES AND TODO LIST:



## Target branch(es):








 
## How to test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here in the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR
